### ISSUE TITLE
Couldn't Reproduce the results.

### ISSUE THREAD

**Issue by mustansarfiaz** [ESandML/FCBFormer#4]
Hi. 

I found your work interesting. Therefore, I decided to work on this topic which is new to me. I'm planning to start some work on medical image segmentation and found your article very interesting.

I tried to reproduce the results for both datasets. For example, train on Kvasir-SEG and test on Kvasir-SEG provided the train/test split by you.  However, I failed to reproduce the results mentioned in the paper. I tried to run with and without pre-train weights. I also tried to re-train the code 2 times, but still failed to reproduce the results. I can reproduce results 2-3% less compared to what was mentioned in the paper. I was using torch 1.10.0 and cuda 11.3.

Could you please let me know any special tricks or library versions or what strategy should I follow to reproduce the results?

Thank you.

**Comment by ESandML** [comment]
Hi mustansarfiaz, thanks for checking out our work!

The reported results were from our first full run with this method, however we have since noticed random variation in the results on repeated runs since we did not seed the initialisation. We did use torch 1.9.0 and cuda 11.1, so you may notice a benefit from using the same versions, but it is more likely to be the random variation in initialisation which causes the difference in results you have observed.
